contracts/payloads/IGP34/PayloadIGP34.sol

```diff
@@ -422,6 +471,62 @@ contract PayloadIGP34 {
         _updateVaultConfig(configs_);
     }
 
+    /// @notice Action 3: Add WBTC vault reward vault on WBTC/USDC and WBTC/USDT
+    function action3() internal {
+        // WBTC/USDC
+        VAULT_T1_FACTORY.setVaultAuth(
+            0x6F72895Cf6904489Bcd862c941c3D02a3eE4f03e, // WBTC/USDC
+            0x4605FC1E6A49D92D97179407E823023F06D5aA0e,
+            true
+        );
+
+        // WBTC/USDT
+        VAULT_T1_FACTORY.setVaultAuth(
+            0xbA379AfC2829CbF5DeA14B8bc135a820e144456D, // WBTC/USDT
```

Comment:
these addresses are the wrong way around 

vault WBTC / USDT: 0x3A0b7c8840D74D39552EF53F586dD8c3d1234C40
new vault rewards: 0xbA379AfC2829CbF5DeA14B8bc135a820e144456D